### PR TITLE
Impostor Client Web version

### PR DESCRIPTION
**Pull request by Igorxp5**
I've created a simple web page to generate ** regionInfo.dat ** for a server such ImpostorClient. My idea is support Android platform using own browser.

I've added a option to add the server IP directly by the link hash (/\#8.8.8.8). So we can start the server and share the link to that with other people: http://aeonlucid.github.io/Impostor/#8.8.8.8

Add that link in the server output may be interesting to easily share :)

**Screenshots:**

**Desktop**
Desktop

**Android**
<p align="center">

</p>

**iOS**
<p align="center">

</p>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="row flex-column justify-content-center align-items-center p-0 m-0">
+    <div class="container text-light d-flex flex-column justify-content-center p-5 p-md-3" style="flex: 1;">
+        <section class="container mb-4">
+            <div class="row justify-content-center">
+                <h1 class="site-title text-center">Impostor</h1>
+            </div>
+            <div class="row justify-content-center">
+                <h2 class="site-subtitle text-center">Amoug Us Private Server</h2>
```
Comment (joaosombrio) [inline index.html:17]: Among*

File: script.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+var REGION_NAME = "Impostor"
+var SERVER_PORT = 22023;
```
Comment (joaosombrio) [inline script.js:2]: Could you make it so you can generate with port? Among us game now supports different ports and the server will too.
I was thinking something like it defaults to 22023 but you can enter a custom one.

**Review by joaosombrio — COMMENTED** [review]
This is such nice work.

**Comment by AeonLucid** [comment]
Awesome stuff, this should be a good "fix" for those on other platforms than Windows. Please change `Amoug` to `Among`. Will also need to rebase this to the `gh-pages` branch when I have created it, but I think I can do that.

**Comment by Igorxp5** [comment]
Server Port field added ;)

Links hash is also supporting server port. Eg: http://aeonlucid.github.io/Impostor/#8.8.8.8:8080

image

**Comment by AeonLucid** [comment]
Thanks! I'll merge this soon.

**Comment by AeonLucid** [comment]
Huh, I tried to rebase this to `gh-pages` and it closed the PR.
image

**Comment by AeonLucid** [comment]
Can you reopen the pull-request @Igorxp5 while targeting the `gh-pages` branch?
Sorry for this.

**Comment by Igorxp5** [comment]
No problem. I've rebased #19 . ;)

**Pull request closed without merging**